vmdb/app/models/ems_microsoft.rb

```diff
@@ -44,10 +44,16 @@ def verify_credentials(_auth_type = nil, options = {})
     raise MiqException::MiqHostError, "No credentials defined" if self.authentication_invalid?(options[:auth_type])
 
     begin
-      run_dos_command("hostname")
+      run_dos_command("powershell import-module virtualmachinemanager; " \
+      "Get-SCVMMServer -ComputerName localhost")
```

Comment:
@Fryguy 
Without the \ the string contains a line break which will prevent Powershell from executing the second part of the command. I can drop the inner closing a opening quote marks to leave this:

    "powershell import-module virtualmachinemanager; \
        Get-SCVMMServer -ComputerName localhost"